src/PrestaShopBundle/Form/Admin/Type/EventListener/PriceReductionListener.php

```diff
@@ -73,6 +87,21 @@ public function adaptReductionField(FormEvent $event): void
                     // Do not forget the row class which is important for JS
                     'class' => 'price-reduction-value',
                 ],
+                'default_empty_data' => 0,
+            ]);
+        // It is possible to have different values in same request, but different events, so if/else is essential
+        // to make sure the form is built as expected during all events
+        } else {
+            $form->add('value', MoneyType::class, [
+                'scale' => $options['scale'],
+                'currency' => $this->currencyDataProvider->getDefaultCurrencyIsoCode(),
+                'attr' => [
+                    'data-currency' => $this->currencyDataProvider->getDefaultCurrencySymbol(),
+                ],
+                'row_attr' => [
+                    'class' => 'price-reduction-value',
+                ],
+                'default_empty_data' => 0,
```

Comment:
Can you also make this generic ? i've seen it a couple of times already :p

Reply:
hmm yeah MoneyType used alot, but slightly different all the time. The only thing that would be 100% reused is the currency option, but I don't feel comfortable introducing those changes in this PR, because there are lots of them, I think it deserves dedicated PR - maybe even one to go through all common forms and try to reuse them (not only money type, because Im pretty sure there are many more like this).